.pre-commit-config.yaml

```diff
@@ -28,19 +16,6 @@ repos:
           - --quiet-level=2
         exclude_types: [csv, json]
         exclude: ^tests/fixtures/|homeassistant/generated/
-  - repo: https://github.com/PyCQA/flake8
-    rev: 6.0.0
-    hooks:
-      - id: flake8
-        additional_dependencies:
-          - pycodestyle==2.10.0
-          - pyflakes==3.0.1
-          - flake8-docstrings==1.6.0
-          - pydocstyle==6.2.3
-          - flake8-comprehensions==3.10.1
-          - flake8-noqa==1.3.0
-          - mccabe==0.7.0
```

Comment:
I don't think ruff can replace McCabe at this point

Reply:
It does: https://github.com/charliermarsh/ruff#mccabe-c90